ultraplot/ticker.py

```diff
@@ -897,3 +1407,141 @@ def __init__(self, *args, **kwargs):
         %(ticker.dms)s
         """
         super().__init__(*args, **kwargs)
+
+
+class CFTimeConverter(mdates.DateConverter):
+    """
+    Converter for cftime.datetime data.
+    """
+
+    standard_unit = "days since 2000-01-01"
+
+    @staticmethod
+    def axisinfo(unit, axis):
+        """Returns the :class:`~matplotlib.units.AxisInfo` for *unit*."""
+        if cftime is None:
+            raise ModuleNotFoundError("cftime is required for CFTimeConverter.")
+
+        if unit is None:
+            calendar, date_unit, date_type = (
+                "standard",
+                CFTimeConverter.standard_unit,
+                getattr(cftime, "DatetimeProlepticGregorian", None),
+            )
+        else:
+            calendar, date_unit, date_type = unit
+
+        majloc = AutoCFDatetimeLocator(calendar=calendar, date_unit=date_unit)
+        majfmt = AutoCFDatetimeFormatter(
+            majloc, calendar=calendar, time_units=date_unit
+        )
+
+        try:
+            if date_type is not None:
+                datemin = date_type(2000, 1, 1)
+                datemax = date_type(2010, 1, 1)
+            else:
+                # Fallback if date_type is None
+                datemin = cftime.DatetimeProlepticGregorian(2000, 1, 1)
+                datemax = cftime.DatetimeProlepticGregorian(2010, 1, 1)
+        except (TypeError, AttributeError):
+            datemin = cftime.DatetimeProlepticGregorian(2000, 1, 1)
+            datemax = cftime.DatetimeProlepticGregorian(2010, 1, 1)
```

Comment:
It seems like these dates are arbitrary and will go out of date in the future. Can we tie them to the current date somehow, or if not, they need to go in the config file.

Reply:
Ignore my comment above. I removed the try block.